Week5/solution-week5.ipynb

```diff
@@ -0,0 +1,144 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "83cc2ca6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "907cff79",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r=[['1','om','bardoli',9081763545],\n",
+    "   ['2','sai','surat',9925464749],\n",
+    "   ['3','ram','mandavi',9934567823],\n",
+    "   ['4','radha','vyara',8867584848],\n",
+    "   ['5','kishan','maharasra',9081764546]];"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "3d7227dd",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "enter student id:6\n",
+      "enter student name:arjun\n",
+      "enter student city:surat\n",
+      "enter student contact:9086758484\n",
+      "enter student id:7\n",
+      "enter student name:bhim\n",
+      "enter student city:vyara\n",
+      "enter student contact:8867564738\n",
+      "enter student id:8\n",
+      "enter student name:sita\n",
+      "enter student city:bardoli\n",
+      "enter student contact:7765875656\n",
+      "enter student id:9\n",
+      "enter student name:laxman\n",
+      "enter student city:mandavi\n",
+      "enter student contact:9988776655\n",
+      "enter student id:10\n",
+      "enter student name:kartik\n",
+      "enter student city:surat\n",
+      "enter student contact:9081564748\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in range(5):\n",
```

Comment:
why you use foor loop?

Reply:
The for loop is used to repeat a section of code known number of times.